scripts/release/run_sourcedeb_job.py

```diff
@@ -45,8 +45,10 @@ def main(argv=sys.argv[1:]):
     add_argument_distribution_repository_key_files(parser)
     parser.add_argument(
         '--source-dir',
-        required=True,
-        help='The directory where the package sources will be stored')
+        required=False,
+        help='DEPRECATED')
+    parser.add_argument(
+        '--sourcepkg-dir', default='/tmp/sourcedeb')
```

Comment:
Since the mount point is `/tmp/sourcedeb`, the argument should reflect that, thus the change from `--source-dir` to `--sourcedeb-dir`.

In reality, the generated Dockerfile only uses the `source` subdirectory, but the `docker run` is mounting the entire `sourcedeb` directory.

Here is the mount: https://github.com/ros-infrastructure/ros_buildfarm/blob/a3820a0b852d8730e98514b5dc4dffe307bf35e6/ros_buildfarm/templates/release/deb/sourcepkg_job.xml.em#L113

Reply:
Once this change has propagated and the argument is being passed, we can drop `--source-dir` and switch `--sourcepkg-dir` to use `add_argument_sourcepkg_dir` and be required.